rocAL/rocAL_pybind/amd/rocal/pipeline.py

```diff
@@ -161,35 +161,33 @@ def rocalRun(self):
         """ Run the pipeline using rocalRun call
         """
         status = b.rocalRun(self._handle)
-        if(status != types.OK):
-            print("Rocal Run failed")
         return status
 
-    def define_graph(self):
+    def defineGraph(self):
         """This function is defined by the user to construct the
         graph of operations for their pipeline.
         It returns a list of outputs created by calling ROCAL Operators."""
-        print("definegraph is deprecated")
+        print("defineGraph is deprecated")
         raise NotImplementedError
 
-    def get_handle(self):
+    def getHandle(self):
         return self._handle
 
-    def GetOneHotEncodedLabels(self, array, device):
-        if device=="cpu":
-            if (isinstance(array,np.ndarray)):
+    def getOneHotEncodedLabels(self, array, device):
+        if device == "cpu":
+            if (isinstance(array, np.ndarray)):
                 b.getOneHotEncodedLabels(self._handle, array.ctypes.data_as(ctypes.c_void_p), self._numOfClasses, 0)
```

Comment:
why the variable self._numOfClasses also in camel case?
It should be in snake case
Please check and change if needed